Server/src/database.ts

```diff
@@ -0,0 +1,41 @@
+import {createConnection} from 'typeorm';
+import {AnimeRank} from './models/entities/AnimeRank';
+
+export const connectDB = async () => {
+  await createConnection().then( async connection => {
```

Comment:
should be indented

Reply:
Which part?

Reply:
await createConnection().then( async connection => {
  console.log("Connection was made.")
  //Methods for Entity Manager: https://github.com/typeorm/typeorm/blob/master/docs/entity-manager-api.md
  // This is using connection.manager.{methodName}